CHANGES.txt

```diff
@@ -39,6 +39,8 @@ Revision 0.2.7, released XX-09-2019
 - Added RFC8419 providing Edwards-Curve Digital Signature Algorithm
   (EdDSA) Signatures in the CMS
 - Added RFC8479 providing Storing Validation Parameters in PKCS#8
+- Added RFC8360 providing Resource Public Key Infrastructure (RPKI)
+  Validation Reconsidered
```

Comment:
This `Validation Reconsidered` sounds funny. I hope that's not a typo!

Reply:
It does sound funny, but that is the title of the RFC...